pkg/server/proto/server.proto

```diff
@@ -652,6 +658,81 @@ message UI {
     // The most recent status report generated for this release.
     StatusReport latest_status_report = 2;
   }
+
+  message ListEventsRequest {
+    // Application that the events belong to.
+    Ref.Application application = 1;
+
+    // Workspace that the events belong to.
+    // If undefined, events from all workspaces will be listed.
+    Ref.Workspace workspace = 2;
+
+    PaginationRequest pagination = 3;
+    SortingRequest sorting = 4;
+  }
+
+  message ListEventsResponse {
+    // The list of events with related messages prefetched.
+    repeated UI.EventBundle events = 1;
+    PaginationResponse pagination = 2;
+  }
+
+  // An event packaged alongside prefetched related messages.
+  message EventBundle {
+    google.protobuf.Timestamp timestamp = 1;
+    EventType eventType  = 2;
+    bytes eventData = 3;
```

Comment:
```suggestion
    EventType event_type  = 2;
    bytes event_data = 3;
```
This was a mistake in the RFC (since corrected) - Should be in snake_case to follow the convention established in the rest of `server.proto`.